ui/component/settingsStripeCard/view.jsx

```diff
@@ -258,10 +282,28 @@ const SettingsStripeCard = (props: Props) => {
   );
 };
 
-export default function Wrapper(props: Props) {
+export default function Wrapper(props: WrapperProps) {
+  const [stripeError, setStripeError] = React.useState();
+  const [stripePromise, setStripePromise] = React.useState(loadStripe(STRIPE_PUBLIC_KEY));
+  const [promisePending, setPromisePending] = React.useState();
+
+  React.useEffect(() => {
+    stripePromise
+      .then(() => setPromisePending(false))
+      .catch((e) => {
+        setPromisePending(false);
+        setStripeError(e.message);
+      });
+  }, [stripePromise]);
+
   return (
     <Elements stripe={stripePromise}>
-      <SettingsStripeCard {...props} />
+      <SettingsStripeCard
+        {...props}
+        promisePending={promisePending}
+        stripeError={stripeError}
+        reloadForm={() => setStripePromise(loadStripe(STRIPE_PUBLIC_KEY))}
+      />
```

Comment:
It did not work for, but perhaps my test method is flawed (do let me know if so).

1. Blocked `stripe.js` from dev tools.
2. The error message and Retry button appears -- nice.
3. Unblock.
4. Press Retry -- expecting another fetch, but his happens (and no fetch):

<img width="442" alt="image" src="https://user-images.githubusercontent.com/64950861/192064638-701ef6ee-cc46-4dca-9639-9c752ad4b147.png">

Reply:
Looks like there's no way to reload the script itself so using a window reload is easier